### ISSUE TITLE
您好，我想问下，假如我语料库改变了，怎么做新的模型啊

### ISSUE THREAD

**Issue by momo1986** [shen1994/ChatRobot#2]
我运行命令：
python data_process.py
得到错误：
![image](https://user-images.githubusercontent.com/34225874/47422757-fee0d500-d7b5-11e8-8553-06ac27600705.png)

**Comment by super1peng** [comment]
这个repo他使用的是自己训练的分词模型，有些字符不能识别，可以替换成常用的分词工具，j如ieba等

**Comment by shen1994** [comment]
yes, you can replace the algorithm 843224306 邮箱：843224306@qq.com 签名由 网易邮箱大师 定制 在2018年12月12日 16:22，lxp 写道： 这个repo他使用的是自己训练的分词模型，有些字符不能识别，可以替换成常用的分词工具，j如ieba等 — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by shen1994** [comment]
@momo1986 use your own TextCut Model, or use mine. And it's better if you have read my code.
